.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,155 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+#  schedule:
+#    - cron:  '0 1 * * MON'
```

Comment:
Is it running the build every Monday? Is this necessary if everything has already passed before the merge?

Reply:
Yeah we rebuild the base image on Friday: https://github.com/votca/buildenv/blob/master/.github/workflows/continuous-integration-workflow.yml#L10, hence we should rebuild this day later or so, but it isn't enabled yet as travis does the cron for now.